src/filesystem/impls/filer/BracketsFiler.js

```diff
@@ -0,0 +1,20 @@
+/*jslint vars: true, plusplus: true, devel: true, nomen: true, regexp: true, indent: 4, maxerr: 50 */
+/*global define, appshell */
```

Comment:
Remove `appshell` here.  Also, you should run `grunt jshint` to see if you are going to have other lint errors.